packages/module/src/SkeletonTable/SkeletonTable.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactNode } from 'react';
+import { Caption, Table, TableProps, TableVariant, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import { Skeleton } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss';
+import { clsx } from 'clsx';
+
+export type SkeletonTableProps = TableProps & {
+  /** Indicates the table variant */
+  variant?: TableVariant;
+  /** The number of rows the skeleton table should contain */
+  rows?: number;
+  /** Any captions that should be added to the table */
+  caption?: ReactNode;
+} & (
+    | {
+        columns: ReactNode[];
+      }
+    | {
+        numberOfColumns: number;
+      }
+  );
+
+const useStyles = createUseStyles({
+  skeleton: {
+    '&.ins-m-dark': {
```

Comment:
this is a dark mode support specific to Red Hat insights, we could probably at least adjust the className to match our rules

Reply:
I think it should be straight-up removed. Pf has its own dark mode support.

Reply:
Was thinking that as well.  Done it's removed.